sumMainSecundaryDiagonal.js

```diff
@@ -0,0 +1,48 @@
+/*let lea = require("prompt-sync")();
+
+var matrix = [
+  [, , , ,],
+  [, , , ,],
+  [, , , ,],
+  [, , , ,],
+];
+
+for (contador = 0; contador < matrix.length; contador++) {
+  let filaMatrix = matrix[contador];
+  let contador2 = 0;
+  console.log("Fila: ", contador);
+  while (contador2 < filaMatrix.length) {
+    let numeroInsertarColumnas = parseInt(lea("Elemento a insertar: "));
+    filaMatrix[contador2] = numeroInsertarColumnas;
+    contador2 = contador2 + 1;
+  }
+}
+console.log(matrix);
+*/
+function sumDiagonals(matrix) {
+  var sumaDiagonalPrincipal = 0;
+  var sumaDiagonalSecundaria = 0;
```

Comment:
this should be declared with let, because declaring it with var will declare the variable globally:
let sumaDiagonalPrincipal = 0;